queue_job/job.py

```diff
@@ -810,6 +832,25 @@ def retryable_example():
             job, default_channel=default_channel, retry_pattern=retry_pattern
         )
 
+    xml_fields = [
+        '    <field name="model_id" ref="[insert model xmlid]" />\n'
+        '    <field name="method">_test_job</field>\n'
+    ]
+    if default_channel:
+        xml_fields.append('    <field name="channel_id" ref="[insert channel xmlid]"/>')
+    if retry_pattern:
+        xml_fields.append('    <field name="retry_pattern">{retry_pattern}</field>')
+
+    xml_record = (
+        '<record id="job_function_[insert model]_{method}"'
+        ' model="queue.job.function">\n' + "\n".join(xml_fields) + "\n</record>"
+    ).format(**{"method": func.__name__, "retry_pattern": retry_pattern})
+    _logger.warning(
```

Comment:
@guewen IMHO this is too much as it will break all runbot builds in repos that use queues. See https://runbot.odoo-community.org/runbot/build/3450323 for example.

Is it wanted ? Or maybe this can go into INFO.

Reply:
You are right, I didn't realize runbot would not agree. We should move it to info, and move the XML snippet to debug because it is too verbose at the moment. I'll try to open a PR tomorrow.